DEHPEcosimPro/ViewModel/Dialogs/DstLoginViewModel.cs

```diff
@@ -123,12 +130,31 @@ public bool RequiresAuthentication
             get => this.requiresAuthentication;
             set => this.RaiseAndSetIfChanged(ref this.requiresAuthentication, value);
         }
-        
+
+        /// <summary>
+        /// Backing field for <see cref="SavedUris"/>
+        /// </summary>
+        private ReactiveList<string> savedUris;
+
+        /// <summary>
+        /// Gets or sets the saved server addresses
+        /// </summary>
+        public ReactiveList<string> SavedUris
```

Comment:
ReactiveList don't need that the private corresponding field is explicitly implemented